src/main/scala/com/scalapenos/riak/internal/RiakHttpClientHelper.scala

```diff
@@ -271,6 +283,29 @@ private[riak] class RiakHttpClientHelper(system: ActorSystem) extends RiakUriSup
       }
     }
   }
+
+  private def toRiakSiblingValues(response: HttpResponse): Set[RiakValue] = {
+    import spray.http._
+    import spray.httpx.unmarshalling._
+    import FixedMultipartContentUnmarshalling._
+
+    implicit val FixedMultipartContentUnmarshaller = multipartContentUnmarshaller(HttpCharsets.`UTF-8`)
+
+    val vclockHeader = response.headers.find(_.is(`X-Riak-Vclock`.toLowerCase)).toList
+
+    response.entity.as[MultipartContent] match {
+      case Left(error) ⇒ throw new ConflictResolutionFailed(error.toString)
+      case Right(multipartContent) ⇒
+        // TODO: make ignoring deleted values optional
```

Comment:
Introduced a global client setting for that. Although the tricky part is testing, as usual. :) Making Riak return these tombstones turned out to be non-trivial (even with riak_kv setting 'delete_mode = keep'). I need to think a bit how to cover it in tests...